features/customer.feature

```diff
@@ -91,3 +91,26 @@ Scenario: Search for john
 #     Then I should see the message "Success"
 #     And I should see "Loki" in the results
 #     And I should not see "fido" in the results
+
+Scenario: Deleting a customer
+    When I visit the "home page"
+    And I set the "first_name" to "sally"
+    And I press the "Search" button
+    Then I should see the message "Success"
+    And I should see "sally" in the "first_name" field
+    When I press the "Delete" button
+    Then I should see the message "Success"
+    And the "customer_Id" field should be empty
+    And the "first_name" field should be empty
+    And the "last_name" field should be empty
+    And the "email" field should be empty
+    And the "password" field should be empty
+    And the "status" field should be empty
+    When I visit the "home page"
+    And I set the "first_name" to "sally"
+    And I press the "search" button
+    Then I should see the message "404 Not Found"
```

Comment:
we don't show a 404 not found message when a search returns nothing. the http call still completes successfully, there is just no search results. So we should look for "success" and then confirm that the table is empty